crates/types/src/rpc/transactions.rs

```diff
@@ -765,11 +824,11 @@ mod tests {
             "nonce_data_availability_mode": "L1",
             "fee_data_availability_mode": "L1",
             "resource_bounds": {
-              "L2_GAS": {
```

Comment:
Why was it capitalized in the first place?

Reply:
it was copied from papyrus. I guess its because they internally use starknet-api, which serializes to uppercase